prefixer/fix-symfony-polyfill.php

```diff
@@ -8,9 +8,16 @@
 $file = __DIR__ . '/../vendor-prefixed/symfony/polyfill-intl-idn/Idn.php';
 $data = file_get_contents($file);
 $data = str_replace('\\Normalizer::', '\\MailPoetVendor\\Normalizer::', $data);
+$data = str_replace('use Normalizer;', 'use MailPoetVendor\\Normalizer;', $data);
```

Comment:
I think that this one is nor necessary since all usages of Normalizer within this file use fully qualified class name. But it makes no harm keeping it.